application/versions.json

```diff
@@ -1,25 +1,129 @@
-adldap2/adldap2                           v10.3.2            A PHP LDAP Package for humans.
-behat/behat                               v3.8.1             Scenario-oriented BDD framework for PHP
-codeception/codeception                   2.4.5              BDD-style testing framework
-codeception/specify                       0.4.6              BDD code blocks for PHPUnit and Codeception
-codeception/verify                        0.4.0              BDD assertion library for PHPUnit
-codemix/yii2-streamlog                    1.3.1              A Yii 2 log target for streams in URL format
-fillup/fake-bower-assets                  2.0.9              Use Composer "replace" to fake out installing bower-asset dependencies ...
-google/apiclient                          v2.9.1             Client library for Google APIs
-google/recaptcha                          1.2.4              Client library for reCAPTCHA, a free service that protects websites fro...
-icawebdesign/hibp-php                     3.1.0              PHP library for accessing the Have I Been Pwned API.
-phpunit/phpunit                           6.5.14             The PHP Unit Testing framework.
-roave/security-advisories                 dev-master 9d9641b Prevents installation of composer packages with known security vulnerab...
-silinternational/email-service-php-client 2.2.0              PHP client to interact with our Email Service API: https://github.com/s...
-silinternational/idp-id-broker-php-client 3.1.0              PHP client to interact with our IdP ID Broker's API: https://github.com...
-silinternational/php-env                  2.1.1              Simple PHP library for getting (or requiring) environment variables, de...
-silinternational/yii2-codeception         dev-master 3fc6968 Patches to implement nice features of yii2-codeception since it was aba...
-silinternational/yii2-email-log-target    1.0.1              Yii2 log target for sending data to email without trace information
-silinternational/yii2-json-log-targets    2.0.0              A collection of Yii2 log targets that format the log message as a JSON ...
-silinternational/zxcvbn-api-client-php    2.0                PHP client library for interacting with Zxcvbn API. See https://github....
-simplesamlphp/saml2                       v3.4.5             SAML2 PHP library from SimpleSAMLphp
-yiisoft/yii2                              2.0.42.1           Yii PHP Framework Version 2
-yiisoft/yii2-debug                        2.1.17             The debugger extension for the Yii framework
-yiisoft/yii2-faker                        2.0.5              Fixture generator. The Faker integration for the Yii framework.
-yiisoft/yii2-gii                          2.2.2              The Gii extension for the Yii framework
-yiisoft/yii2-swiftmailer                  2.1.2              The SwiftMailer integration for the Yii framework
+{
```

Comment:
The only actual version change in this (since the reformatting hides the meaningful changes) is to roave/security-advisories.